design/API.yml

```diff
@@ -1595,7 +1760,7 @@ paths:
               properties:
                 content:
                   type: string
-                quote_id:
+                reply_id:

```

Comment:
no need for reply_id. Only include insights API definition in this pr, while comments API changes shall be part of the comments pr #4